src/com/cdd/bao/util/URITextFieldDelegate.java

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
The functionality in this class isn't enough to justify having its own separate file; make it into an inner class instead.

Reply:
Its an interface.

Reply:
That's fine: interfaces can be inner classes. (Making a separate class for line + button widget is already a bit of a stretch, but adding 2 new files is definitively over the line.)